internal/action/show.go

```diff
@@ -218,7 +218,9 @@ func (s *Action) showGetContent(ctx context.Context, sec gopass.Secret) (string,
 				}
 				sb.WriteString(strings.Join(v, "\n"+k+": "))
 			}
-			sb.WriteString("\n")
+			if sec.Body() != "" || l < len(sec.Keys())-1 {
```

Comment:
Ok, I get what you're doing but I think this is suble enough to ask for a short comment.